docs/04-guides/on-demand/update-video-asset/02-api.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: api
+title: API
+---
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get them 
+ready for optimized playback. Videos can be provided either by you (static content) 
+or your users, given your application offers an interface for them to do so. 
+
+You should be familiar with [Assets](/reference/api/get-assets) and [Tasks](/reference/api/get-tasks) 
+and have generated a [Livepeer Studio API key](/quickstart/).
+
+## Step 1: Get the `asset.id` of an existing asset
+Get the `asset.id` of an existing asset by making a request to list all existing assets in your account.
+If you haven't created an asset yet, you can follow the [upload a video asset guide](/guides/on-demand/upload-a-video-asset#api) to do so.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="Node" label="Node">
+
+```js
+const getAssets = await fetch("https://livepeer.studio/api/asset", {
+    method: "GET",
+    headers: {
+        Authorization: `Bearer ${process.env.API_TOKEN}`,
+        "Content-Type": "application/json",
+    }
+});
+```
+
+</CodeBlockTabItem>
+<CodeBlockTabItem value="curl" label="curl">
+
+```bash
+curl --location --request GET 'https://livepeer.studio/api/asset' \
+--header 'Authorization: Bearer $API_TOKEN'
+```
+
+</CodeBlockTabItem>
+</CodeBlockTabs>
+
+## Step 2: Update the asset
+Once you have an `asset.id`, you can make a request to update the asset's storage location.
```

Comment:
The code example is showing up to update an asset's name and metadata not its location.